src/components/Viewer.js

```diff
@@ -1,8 +1,10 @@
 import {Verses} from "scripture-resources-rcl";
 import usfmJS from 'usfm-js';
+import {tokenize} from 'string-punctuation-tokenizer';
 
 import { parseResourceLink, extendProject, resourceFromResourceLink, getResourceManifest } from 'scripture-resources-rcl';
 import { versesFromReferenceIdAndBooks, referenceIdFromReference } from 'scripture-resources-rcl';
+import Card from '../components/Card';
```

Comment:
Change this to @components/Card, which is defined in the next.js config jsconfig.json (very handy)